src/integrationTest/resources/application.yaml

```diff
@@ -163,8 +163,22 @@ idam:
     url: https://idam-api.aat.platform.hmcts.net
   client:
     id: prl-cos-api
+    baseUrl: https://idam-api.aat.platform.hmcts.net
     redirect_uri: https://manage-case.aat.platform.hmcts.net/oauth2/callback
     secret: ${IDAM_CLIENT_SECRET:xxxxxxx}
+    clientId: xuiwebapp
```

Comment:
are these changes required?

Reply:
Not particularly but added in the process of resolving innitial build issues to mimmick other test application.yaml files